### PR TITLE
Feat/mg/ssl self signed fix

### PR DESCRIPTION
**Pull request by yelodevopsi**

### CODE REVIEW

File: backend/appsettings.Development.json [hunk 1]
```diff
@@ -4,5 +4,8 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "ConnectionStrings": {
+    "VibesDb": "Server=localhost;Database=AzureVibesDb;User Id=sa;Password=yourStrong(!)Password;;TrustServerCertificate=true"
```
Comment (yelodevopsi) [inline backend/appsettings.Development.json:9]: @jonasbjoralt What was needed here again for dev-mode?

Comment (jonasbjoralt) [reply backend/appsettings.Development.json:9]: This is fine I think

File: docker-compose.yml [hunk 2]
```diff
@@ -5,18 +5,17 @@ services:
     build:
       context: .
       dockerfile: ./frontend/Dockerfile
-    container_name: vibes-yay
+    container_name: vibes-frontend-docker
```
Comment (jonasbjoralt) [inline docker-compose.yml:8]:
```suggestion
    container_name: vibes-frontend
```
"docker" is somewhat redundant. not a big issue though

**Review by jonasbjoralt — APPROVED** [review]

**Review by yelodevopsi — COMMENTED** [review]
Good. All good here.

**Pull request merged**